devlab/generate_load.py

```diff
@@ -217,7 +193,8 @@ def _get_body_for_image_creating(_image):
             img = self.glanceclient.images.create(
                 **_get_body_for_image_creating(image))
             img_ids.append(img.id)
-        wait_until_images_created(img_ids[:])
+        self.wait_until_objects_created(img_ids[:], self.check_image_state,
```

Comment:
I will re-use the same image list only in this code block. In other cases i don't need to save lists like this.